server.go

```diff
@@ -46,7 +47,7 @@ const (
 )
 
 func main() {
-	logrus.SetLevel(logrus.InfoLevel)
+	logrus.SetLevel(logrus.DebugLevel)
```

Comment:
Should we put this back to InfoLevel? O is it useful to debug the feature when testing the frontend integration?

Reply:
Good call